### ISSUE TITLE
Country names can be consistent across the data

### ISSUE THREAD

**Issue by rmunjuluri** [CSSEGISandData/COVID-19#1214]
Hello,

I see that the country names sometimes use alternate names creating a view that appears to show multiple countries. If the data can use consistent names across the time-series... it will be a lot cleaner and leaner. For example, I am having to create a duplicate entry table to dedup the country names.

Duplicate   Entry | Country
-- | --
North Ireland | UK
Iran (Islamic Republic of) | Iran
Taipei and environs | Taiwan
Macao SAR | Macau
Holy See | Vatican City
Taiwan* | Taiwan
Cote d'Ivoire | Ivory Coast
Republic of Ireland | Ireland
Republic of Korea | South Korea
Viet Nam | Vietnam
Russian Federation | Russia
Korea, South | South Korea
Moldova | Republic of Moldova
Hong Kong SAR | Hong Kong
occupied Palestinian territory | Palestine
Saint Martin | St. Martin
Cruise Ship | Others
Reunion | France
Channel Islands | UK
United Kingdom | UK
Czechia | Czech Republic
Mainland China | China
Macao | Macau
Gambia, The | The Gambia
Bahamas, The | Bahamas
Cape Verde | Cabo Verde
Timor-Leste | East Timor

Please let me know if I can help with this and I can post the duplicate names everyday if required

**Comment by mhannigan** [comment]
Yes, this should be done. It is far more consistent.

**Comment by neuroactive** [comment]
I agree the names need cleaning up. However, the name of the country should be "United Kingdom" and not the abbreviated form "UK". Also, "Saint Martin" should be written in full rather than abbreviated.

**Comment by neuroactive** [comment]
Also, Moldova is fine as a country name. It would be inconsistent to use "Republic of Moldova", but not "Republic of Ireland", and likewise for many other countries.

**Comment by rmunjuluri** [comment]
Absolutely agree neuroactive ! I will fix these.

**Comment by neuroactive** [comment]
I don't believe the Channel Islands are technically part of the UK. See: https://en.wikipedia.org/wiki/Channel_Islands
"They are considered the remnants of the Duchy of Normandy and, although they are not part of the United Kingdom, the UK is responsible for the defence and international relations of the islands."

**Comment by rmunjuluri** [comment]
I am trying to find the UN 3-Alpha code for Channel Islands. Any idea of what it is ? I will need that to merge with geo-data that keys of the UN 3-Alpha code.

**Comment by rmunjuluri** [comment]
also I realized that the latest time series shows St. Martin as a state/province under France.  I must have picked St. Martin as its own country with a unique UN 3-Alpha code. I have rolled it up under France now.

**Comment by neuroactive** [comment]
Jersey and Guernsey have their own codes, I believe.

**Comment by rkaseler** [comment]
The Channel Islands includes Guernsey and Jersey, each of which has codes of its own, per ISO 3166. See: https://en.m.wikipedia.org/wiki/List_of_ISO_3166_country_codes

**Comment by rkaseler** [comment]
Suggest adapting the data structure to conform to ISO 3166-1, 3166-2 and 3166-3 and then normalizing existing country data to uniquely use the country names, numeric codes from the standard. The Alpha 3-codes could be use for graphical display.
https://en.m.wikipedia.org/wiki/List_of_ISO_3166_country_codes

**Comment by MhdHejazi** [comment]
I collected all the country names I found in the data in one file and added the ISO code for every country. I'm using this file to convert the non-standard country names into standard ISO codes, and then I can get the localized name for every country code from the system.

You can find the [file here](https://github.com/MhdHejazi/CoronaTracker/blob/master/Corona/Resources/ISO-Country.yml).

**Comment by rollick** [comment]
@mhdhejazi They are suggesting they will soon start including the ISO code - https://github.com/CSSEGISandData/COVID-19/issues/1250

This is a good start and will hopefully push them to use the official names.
